src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.js

```diff
@@ -245,12 +253,15 @@ export class StepIndexPatternComponent extends Component {
       const errorMessage = intl.formatMessage(
         {
           id: 'kbn.management.createIndexPattern.step.invalidCharactersErrorMessage',
-          defaultMessage: 'An index pattern cannot contain spaces or the characters: {characterList}'
+          defaultMessage: 'A ${indexPatternName} cannot contain spaces or the characters: {characterList}'
```

Comment:
You're right. Though I'm concerned about how this will affect translators. Implementing this would require creating a `indexPatternTypeArticle` on each index pattern creation config instance, e.g.

```js
const indexPatternTypeArticle = i18n.translate('common.ui.management.editIndexPattern.createIndex.defaultTypeArticle',
  { defaultMessage: 'An index pattern' });
```

This would then be interpolated into the various messages where the article is needed. I'm concerned about how this level of interpolation will complicate translations. Translators won't have the end context available when defining the article's translation, so the end result may not make sense. For that reason, I think we should punt on this. What do you think, should we punt?